support-frontend/assets/components/checkoutForm/formSectionHiddenUntilSelected.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+// eslint-disable-next-line no-unused-vars
+import React, { type Node } from 'react';
+/** @jsx jsx */ import { jsx, css } from '@emotion/core';
+import { headline } from '@guardian/src-foundations/typography';
+import { palette } from '@guardian/src-foundations';
```

Comment:
We've found that importing the entire palette can bloat your JS bundle as you're loading lots of colours that you're not using. We recommend importing only the colours for the context in which you're using them from the `/palette` folder:

```ts
import { border } from '@guardian/src-foundations/palette'

const showFormSection = css`
    border-top: 1px solid ${border.secondary};
`;
```

Reply:
Thanks!